types.go

```diff
@@ -251,20 +280,48 @@ type Repo struct {
 	GithubType     string `json:"githubType,omitempty"`
 	InheritedCreds bool   `json:"inheritedCreds,omitempty"`
 }
-type HarnessRepository struct {
-	Repo `json:"repo"`
-}
-
-type UpdateMask struct {
-	Paths []string `json:"Paths"`
-}
-
 type RepoWithUpdateMask struct {
 	Repo       `json:"repo"`
 	UpdateMask struct {
 		Paths []string `json:"paths"`
 	} `json:"updateMask"`
 }
+type UpdateMask struct {
```

Comment:
I don't think `UpdateMask` is required anymore, since it's being declared internally in line# 285.